packages/formation-react/src/components/Pagination/Pagination.jsx

```diff
@@ -8,25 +8,50 @@ class Pagination extends Component {
   static propTypes = {
     ariaLabelSuffix: PropTypes.string,
     className: PropTypes.string,
+    enableAnaltyics: PropTypes.bool,
     maxPageListLength: PropTypes.number.isRequired,
     onPageSelect: PropTypes.func.isRequired,
     page: PropTypes.number.isRequired,
     pages: PropTypes.number.isRequired,
     showLastPage: PropTypes.bool,
+    trackEvent: PropTypes.func,
   };
 
   static defaultProps = {
     ariaLabelSuffix: '',
+    enableAnaltyics: true,
```

Comment:
Typo I think with the 'y' and the 't' - `enableAnaltyics`

Reply:
Nice catch! If it's all spelled wrong, it works perfectly! 😂

Reply:
Haha well said!